packages/@lightningjs/ui-components/src/components/Card/CardTitle.d.ts

```diff
@@ -19,21 +19,46 @@
 import lng from '@lightningjs/core';
 import Card, { CardStyle } from './Card';
 import { TextBoxStyle } from '../TextBox';
-import type { TextContent } from '../InlineContent/InlineContent';
-import type { StylePartial } from '../../types/lui';
+import { TextContent } from '../InlineContent/InlineContent';
+import { StylePartial } from '../../types/lui';
 
-export type CardTitleStyle = CardStyle & {
-  descriptionTextProperties: TextBoxStyle;
+type CardTitleStyle = CardStyle & {
+  descriptionTextStyle: TextBoxStyle;
   detailsTextProperties: TextBoxStyle;
 };
 
-export default class CardTitle extends Card {
+declare namespace CardTitle {
+  export interface TemplateSpec extends Card.TemplateSpec {
+    /**
+     * description???
+     */
+    description?: string | TextContent[];
+    /**
+     * details??
+     */
```

Comment:
lets copy over the comments from the mdx API table

Reply:
yep.  I should have triple checked this PR before saying it was ready.